Describe QGIS Server properties for raster and WM(T)S layers (#8008)
qgis/QGIS-Documentation: docs/user_manual/working_with_raster/raster_properties.rst

QGIS Server Properties
======================

The |overlay| :guilabel:`QGIS Server` tab helps you configure

> settings of the data when published on a server.

Reviewer: Maybe be more specific here that all these properties are related to publishing by QGIS Server only and not on a generic OGC server? It is clear to me but I am wondering if this would generate confusion in the "average" QGIS Desktop users that usually do not know about QGIS server at all.

Author: ```suggestion
settings of the data when published by :ref:`QGIS Server <QGIS-Server-manual>`.
```

The configuration concerns:

* :guilabel:`Description`: provides information to describe the data,
  such as :guilabel:`Short name`, :guilabel:`Title`, :guilabel:`Summary`,
  a :guilabel:`List of Keywords`,  and a :guilabel:`Data URL`
  whose :guilabel:`Type` can be in ``text/html``, ``text/plain`` or ``application/pdf``.
* :guilabel:`Attribution`: a :guilabel:`Title` and :guilabel:`URL`
  to identify who provides the data
* :guilabel:`Metadata URL`: a list of :guilabel:`URL` for the metadata
  that can be of ``FGDC`` or ``TC211`` :guilabel:`Type`,
  and in ``text/plain`` or ``text/xml`` :guilabel:`Format`
* :guilabel:`Legend URL`: a :guilabel:`URL` for the legend,
  in either ``image/png`` or ``image/jpeg`` :guilabel:`Format`

> 

Author: ```suggestion

.. note::
  When the raster layer you want to publish is already provided by a web service,
  further :ref:`properties <wms_server_properties>` are available for setting.

```
